src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Network/ApplicationGateway/Add-ArmApplicationGatewayBackendHttpSettings.ps1

```diff
@@ -23,7 +23,9 @@ function Add-ArmApplicationGatewayBackendHttpSettings {
         [int]
         $RequestTimeoutInSec = 30,
         [string]
-        $TrustedRootCertificateName
+        $TrustedRootCertificateName,
+        [PSCustomObject]
+        $Probe
```

Comment:
Isn't it the probe name here?

Reply:
In the template it looks like 
`"probe" : { "id": "..."}`
The probe name is not enough to get its id. We also need the app gateway `_resourceId`
I can change it to `$ProbeId` instead